experimental/devices/pca9685/pca9685.go

```diff
@@ -111,40 +110,29 @@ func (d *Dev) SetPwmFreq(freqHz physic.Frequency) error {
 
 	time.Sleep(100 * time.Millisecond)
 
-	_, err := d.dev.Write([]byte{mode1, (byte)(oldmode | 0x80)})
+	_, err := d.dev.Write([]byte{mode1, oldmode | restart})
 	return err
 }
 
-// SetAllPwm set a pwm value for all outputs
+// setPWM writes a PWM value in a specific register.
+func (d *Dev) setPwm(register uint8, on, off gpio.Duty) error {
```

Comment:
Update the function name too, not just the comment.

Let's punt on the public ones for now to not break clients yet, but the private one can be fixed right away.

Reply:
That was an oversight on my part, sorry.